### ISSUE TITLE
Level 2: Contact Us Page

### ISSUE THREAD

**Issue by hannahlivnat** [hannahlivnat/Ask-EDD-Project#14]
## This Issue is Complete When: 

- [ ]  Each contact div is a reloadable component which takes a unique: icon, header, subheader, button

- [ ] button is a reloadable component stored in the "common" folder within the component folder

- [ ] On click, there is an event listener that currently logs "button clicked" on console

## Figma Mockup: 

https://www.figma.com/file/SYCaUO0bX527hsjRt1ZioC/EDD-Help-Page?node-id=24%3A407

## Screenshots: 

![Screen Shot 2020-09-27 at 12 14 24 PM](https://user-images.githubusercontent.com/54752231/94373560-018f8780-00bb-11eb-8592-8cd242139c4a.png)

**Comment by utk0730** [comment]
Hi Hannah,
I would like to work on this issue to start with the codebase. And hopefully more contributions to this project going forward

**Comment by hannahlivnat** [comment]
Hey Utkarsh,

Sounds great! I'm working on getting the data structure for content set up
currently, so issues #2 (Navigation), #10 (Header), #11(Footer) would be
great to start with if you're comfortable with Sass. Let me know if
you'dl like to take one of those issues on, and I'll move it to in progress
on the project board.

Hannah

**Comment by utk0730** [comment]
yeah sure , I can start with the prioritised issues. Please assign me these

**Comment by hannahlivnat** [comment]
I'll move these over to the in progress! Let me know if you have any
difficulty accessing the issues or the Figma file.

https://github.com/hannahlivnat/Ask-EDD-Project/issues/2
https://github.com/hannahlivnat/Ask-EDD-Project/issues/10
https://github.com/hannahlivnat/Ask-EDD-Project/issues/11

Hannah

**Comment by utk0730** [comment]
Sure thanks

**Comment by utk0730** [comment]
@hannahlivnat May I pick this one next ?

**Comment by hannahlivnat** [comment]
@utk0730  Sure, I assigned it to you!
